lib/checks/mobile/css-orientation-lock.js

```diff
@@ -0,0 +1,135 @@
+/* global context */
+
+// extract asset of type `cssom` from context
+const { cssom = undefined } = context || {};
+const checkPass = true;
+const checkFail = false;
+const checkIncomplete = undefined;
```

Comment:
This return value thing is inconsistent with how we've done things so far. I'm open to have a discussion about it, but I don't think we should be introducing inconsistent coding styles like this.

Reply:
Agree. Not a big thing. Will change in a separate PR.